src/main/java/com/gluonhq/substrate/target/AbstractTargetConfiguration.java

```diff
@@ -978,6 +979,7 @@ List<String> getTargetSpecificAOTCompileFlags() throws IOException {
         return Collections.emptyList();
     }
 
+    @Deprecated
```

Comment:
Why is deprecated? Is there a replacement for it?

Reply:
No, LLVM backend is not really be maintained, and we don't use it anymore (or we should not).
I don't like removing public methods without deprecating them first, so the annotation here is a warning that this should not be used anymore.

Reply:
Ok, makes sense
Then maybe:
`InternalProjectConfiguration.isUseLLVM()` and `InternalProjectConfiguration.canRunLLVM()` should be deprecated too?

Reply:
ok